src/lib/image.js

```diff
@@ -12,17 +12,21 @@ export async function downloadImage (url) {
   return response
 }
 
-export async function identify (buffer) {
+export async function identify (buffer, supportWebp) {
```

Comment:
Maybe we should set the default value to `false` for `supportWebp` in all these functions? In case dev updated vsf-api but did not update config?

Another proposal is replacing `supportWebp` to `options: ImageOptions` spreaded object where:
```ts
interface ImageOptions {
  supportWebp: boolean;
  // In future we might want new things, and we should be able to so easy omit Breaking Changes when it is object, e.g. supportAvif: boolean; 
}
```